src/main/java/com/google/api/codegen/metacode/InitFieldConfig.java

```diff
@@ -67,4 +69,9 @@ public boolean isFormattedConfig() {
   public boolean hasFormattedInitValue() {
     return entityName() != null && value() != null;
   }
+
+  private static String parseValueString(String valueString) {
+    String randomValue = Integer.toString(Math.abs(valueString.hashCode()));
```

Comment:
Good point. I changed the hash string so the $RANDOM value would be more unique. It is not a strict requirements that each $RANDOM value needs to be unique. But it should be unique enough so the data can be validated by tests.